test_suite/tests/ui/default-attribute/enum_path.stderr

```diff
@@ -1,5 +1,5 @@
 error: #[serde(default = "...")] can only be used on structs with named fields
- --> tests/ui/default-attribute/enum_path.rs:5:1
+ --> tests/ui/default-attribute/enum_path.rs:4:9
   |
-5 | enum E {
-  | ^^^^
+4 | #[serde(default = "default_e")]
+  |         ^^^^^^^
```

Comment:
I am not sure whether to prefer underlining just the ident or the whole nested attribute. Do you have an opinion?

```console
#[serde(before, default = "...", after)]
                ^^^^^^^
```

```console
#[serde(before, default = "...", after)]
                ^^^^^^^^^^^^^^^
```

I think the second behavior is implementable with `cx.syn_error(meta.error(msg))`.

Reply:
I guess the latter is a bit nicer, thanks for the impl hint.